packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md

```diff
@@ -30,21 +31,61 @@ This guide explains how to customize the sign-in page for both redirect and embe
 
 ## About the sign-in page
 
-The sign-in page is a JavaScript library that gives you a fully-featured and customizable sign-in experience that you can use to authenticate users on any website. How you customize the sign-in page depends on whether Okta is hosting it (redirect authentication) or you are embedding it in your app (embedded authentication).
+The sign-in page is a JavaScript library that gives you a fully featured and customizable sign-in experience that you can use to authenticate users on any website. How you customize the sign-in page depends on whether Okta is hosting it (redirect authentication) or you're embedding it in your app (embedded authentication).
 
 * Redirect authentication: Okta hosts the sign-in page that appears when your applications redirect to Okta to sign users in. You can customize the page using easy controls or a code editor that is provided. See [Style for redirect authentication](#style-for-redirect-authentication).
 
-* Embedded authentication: After you have [installed the Okta Sign-In Widget](https://github.com/okta/okta-signin-widget#embedded-self-hosted) into your project and configured the authentication scenarios that you want to support, you can then customize the sign-in page. You can apply customizations to match your branding using CSS and JavaScript. See [Style for embedded authentication](#style-for-embedded-authentication).
+* Embedded authentication: After you've [installed the Okta Sign-In Widget](https://github.com/okta/okta-signin-widget#embedded-self-hosted) into your project, configure the authentication scenarios that you want to support. Then you can customize the sign-in page. Apply customizations to match your branding using CSS and JavaScript. See [Style for embedded authentication](#style-for-embedded-authentication).
 
 ## Style for redirect authentication
 
-You can add any HTML, CSS, or JavaScript to the sign-in page and also customize it [per application](#per-application-customization) and with multiple brands. This page covers what you can change when you are using redirect authentication, how to use the variables and request context, and also how to bypass the custom sign-in page.
+You can add any HTML, CSS, or JavaScript to the sign-in page and also customize it [per application](#per-application-customization) and with multiple brands. You can use variables and request context, and bypass the custom sign-in page.
```

Comment:
Maybe simplify to "You can add HTML, CSS, and JavaScript..."? 

I think 'per' may fall under 'Latin terms' so you may want to instead use "customize it for each application" (may not apply to devdocs; per is clear to me, I just am uncertain if it violates our style)

Would the entire first sentence look better as:

You can add HTML, CSS, and JavaScript to the sign-in page, customize it [per application] and for multiple brands."

I'm confused by the 'request context' bit; but that may be a knowledge shortcoming on my part.